src/miner.cpp

```diff
@@ -744,9 +750,31 @@ void NavCoinStaker(const CChainParams& chainparams)
                 MilliSleep(1000);
             }
 
+            if (nLastTime != 0 && nLastSteadyTime != 0)
```

Comment:
Should this be split out to a function? - just a question not saying we should

Reply:
i don't reject the idea